src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  // Rozdzielamy ciąg na linie, usuwamy puste linie i linie zawierające tylko białe znaki
+  const lines = sourceString.split(';').filter(line => line.trim());
```

Comment:
The chaining of methods should follow the indentation style as per the checklist item 4. Each chained method should be on a new line with the same indentation to improve readability.